apps/client/src/app/modules/list/list-details-panel/list-details-panel.component.ts

```diff
@@ -455,6 +488,30 @@ export class ListDetailsPanelComponent implements OnChanges, OnInit {
     this.message.success(this.translate.instant('LIST.Copied_as_text'));
   }
 
+  public copyJSONExport() {
+    let rows: ListRow[];
+    if (this.tiers) {
+      rows = this.tiers.reduce((res, tier) => {
+        return [...res, ...tier];
+      }, []);
+    } else {
+      rows = this.displayRow.rows;
+    };
+
+    if (this._clipboardService.copyFromContent(
+      JSON.stringify(this.getSerializedRowData(rows))//,null,2)
+    ))
+      this.jsonCopied();
+  }
+
+  private getSerializedRowData(rows: ListRow[]): any {
+    return this.serializationHelper.getJsonExport(UniversalisService.GetDCFromServerName(this.lazyData.datacenters, this.server$.getValue()), this.server$.getValue(), rows);
```

Comment:
You can't use a behaviorSubject just to have a sync value, if you want to be able to use the server as value, make it a function parameter and inject it from the template using the async pipe.